src/Akeneo/Category/front/src/feature/pages/CategoryEditPage.tsx

```diff
@@ -62,11 +64,22 @@ const CategoryEditPage: FC = () => {
   const [tree, setTree] = useState<EnrichCategory | null>(null);
 
   // ui state
-  const [activeTab, setActiveTab] = useSessionStorageState(propertyTabName, 'pim_category_activeTab');
+  const [activeTab, setActiveTab] = useSessionStorageState<string>(
+    isGranted('pim_enrich_product_category_edit_attributes') ? Tabs.ATTRIBUTE : Tabs.PROPERTY,
+    'pim_category_activeTab'
+  );
   const [isCurrent, switchTo] = useTabBar(activeTab);
   const [secondaryActionIsOpen, openSecondaryAction, closeSecondaryAction] = useBooleanState(false);
   const [isDeleteCategoryModalOpen, openDeleteCategoryModal, closeDeleteCategoryModal] = useBooleanState();
 
+  const handleSwitchTo = useCallback(
+    (tab: string) => {
+      setActiveTab(tab);
+      switchTo(tab);
+    },
+    [setActiveTab, switchTo]
```

Comment:
The `setActiveTab` and `switchTo` functions don't seem to be stateful. IMHO, they are initialized at the first execution and we do not need to define them in the key of this `useCallback`. It should avoid unexpected rerender.

```suggestion
    []
```

Reply:
Same as above